MetabolicAtlas/MetabolicAtlas
# feat: enzyme landing page

**Related issue(s) and PR(s)**  
This PR closes #1015 and relates to #1014.

This is joint work with @e0 .


**Type of change**  


 
**List of changes made**  

- Added the page `/gotenzymes`
- Added "GotEnzymes" to the navbar
- Added search field and example search strings
- Added about text

**Screenshot of the fix**  

<img width="1523" alt="image" src="https://user-images.githubusercontent.com/423498/174041346-6c041036-823e-4681-924b-7f9c860329fd.png">

**Testing**  

- Navigate to `/gotenzymes` and try out the provided example searches
- Please also try other searches. You may search for EC, reaction and compound ids. Examples: "C00002", "2.5.1.18", "R00006"
- Searching for non-valid strings:
  -  reaction like, compound like or ec like strings (eg "RRR", "CCC", "2222.2.2.2") should send you to the respective page, with an error message saying that nothing was found.
  - for other search strings (eg "xxxx"), "No matches for xxxx" will be shown"
- Verify that the about text section looks ok and that the links work.

**Further comments**  

This will be undrafted when #1030  is merged.

**Definition of Done checklist**
